app/src/main/java/org/eyeseetea/malariacare/network/ServerAPIController.java

```diff
@@ -726,6 +727,94 @@ public static boolean isUserClosed(String userUid) {
         return closedDate.before(new Date());
     }
 
+    public static org.eyeseetea.malariacare.domain.entity.OrgUnit getOrganisationUnitsByCode(
```

Comment:
@manuelplazaspalacio After @ifoche merge this PR https://github.com/EyeSeeTea/pictureapp/pull/1074 There is already a method getCurrentOrgUnit with similar logic, can you refactor for unify code?